contracts/state.cairo

```diff
@@ -360,6 +368,51 @@ func get_pool_volatility_auto{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, ra
 }
 
 
+@view
+func get_max_lpool_balance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    pooled_token_addr: Address) -> (max_balance: Uint256) {
+        let(maxbal) = max_lpool_balance.read(pooled_token_addr);
+        return (maxbal, );
+}
+
+// External here is just to make sure testnet's migration can go through correctly. Will be DEPRICATED later
+@external
+func set_max_lpool_balance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    pooled_token_addr: Address, max_lpool_bal: Uint256) {
+
+        with_attr error_message("Max lpool balance can be set only by admin"){
+            Proxy.assert_only_admin();
+        }
+    
+        with_attr error_message("max lpool balance can't be negative") {
+            assert_uint256_le(Uint256(0, 0), max_lpool_bal);
+        }
+        max_lpool_balance.write(pooled_token_addr, max_lpool_bal);
+        return ();
+}
+
+@external
+func set_max_opt_size_perc{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
`set_max_option_size_percent_of_voladjspd` the getters and setters should be `get_exact_storage_var_name` and `set_exact_storage_var_name`